packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -601,36 +601,80 @@ class _CupertinoContextMenuState extends State<CupertinoContextMenu> with Ticker
   }
 }
 
-// A floating copy of the CupertinoContextMenu's child.
-//
-// When the child is pressed, but before the CupertinoContextMenu opens, it does
-// an animation where it slowly grows. This is implemented by hiding the
-// original child and placing _DecoyChild on top of it in an Overlay. The use of
-// an Overlay allows the _DecoyChild to appear on top of siblings of the
-// original child.
-class _DecoyChild extends StatefulWidget {
-  const _DecoyChild({
+
+/// A floating copy of the CupertinoContextMenu's child.
+///
+/// When the child is pressed, but before the CupertinoContextMenu opens, it does
+/// an animation where it slowly grows. This is implemented by hiding the
+/// original child and placing DecoyChild on top of it in an Overlay. The use of
+/// an Overlay allows the DecoyChild to appear on top of siblings of the
+/// original child.
+class DecoyChild extends StatefulWidget {
```

Comment:
Whoo I never meant for something named "DecoyChild" to end up public haha. I think you can still do your test without making this public. Find the _DecoyChild by runtimeType, find the Container descendant with the BoxDecoration, then read the boxDecoration property.

Reply:
Thanks, this was very helpful in cleaning up a lot of mess I made before 😅, also thanks to @navaronbracke, he also hinted at this but I proceeded with my way, now I realize.